data/settings/src/main/proto/com/google/jetpackcamera/settings/capture_mode.proto

```diff
@@ -20,6 +20,7 @@ option java_package = "com.google.jetpackcamera.settings";
 option java_multiple_files = true;
 
 enum CaptureModeProto {
```

Comment:
We should rename this to `CaptureMode` protos shouldn't contain "Proto" in the name.

When we need to differentiate them where we use them, we can potentially use import as aliases.